changelog/886.bugfix.rst

```diff
@@ -0,0 +1 @@
+Allowed eV inputs for temperatures without conversion warnings.
```

Comment:
This statement should be generalized, because the change you're making affects all unit conversions that utilize an `equivalencies`.  It won't just affect `eV -> K`, but it'll also affect `C -> K`.

Reply:
True, it was written with the goal of the PR in mind rather than the eventual side effects.